client/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <title>Tenthop - Glamp In Style</title>
+  </head>
+  <body>
+    <div id="photo-module"></div>
+    <div id="photo-carousel-modal"></div>
+    <script src="http://127.0.0.1:3001/bundle.js" crossorigin></script>
```

Comment:
I don't think you need the crossorigin in your script-- what does it do?

Reply:
Doesn't it make crossorigin requests?  I'll take it out if needed.